WordPress/Classes/ViewRelated/Site Creation/Final Assembly/SiteAssemblyWizardContent.swift

```diff
@@ -303,12 +307,14 @@ extension SiteAssemblyWizardContent: NUXButtonViewControllerDelegate {
         }
 
         dismissTapped(viaDone: true) { [blog, weak self] in
+            RootViewCoordinator.shared.isSiteCreationActive = false
             RootViewCoordinator.sharedPresenter.showBlogDetails(for: blog)
+            RootViewCoordinator.shared.reloadUIIfNeeded(blog: blog)
```

Comment:
Moving the reload trigger outside the completion fixes the glitch of showing an invalid state for a split second before reloading.
```suggestion
        RootViewCoordinator.shared.isSiteCreationActive = false
        RootViewCoordinator.shared.reloadUIIfNeeded(blog: blog)
        
        dismissTapped(viaDone: true) { [blog, weak self] in
            RootViewCoordinator.sharedPresenter.showBlogDetails(for: blog)
```